### ISSUE TITLE
triangles with gradient

### ISSUE THREAD

**Issue by jadsongmatos** [RH12503/Triangula#9]
it would be possible to use overlapping triangles with gradient to get different results

**Comment by rh12503** [comment]
Could you please elaborate on your desired effect? I'm trying to keep Triangula less bloated, so you might also want to consider using a vector editor (eg. Illustrator) instead if possible.

**Comment by jadsongmatos** [comment]
I made this demonstration of a triangle with gradient, where in the image on the left we have the test image converted to opaque triangles and in the image on the right we have the image of the test with gradient triangle

![bitmap](https://user-images.githubusercontent.com/50336793/115978031-4d23ea00-a553-11eb-8ec8-19f3740cc0fe.png)

here you have a demonstration done manually in inkscape

![Screenshot_20210424_231057](https://user-images.githubusercontent.com/50336793/115978050-78a6d480-a553-11eb-8a03-ca4a40559392.png)

I chose to do a test with a black and white image because I thought it would be easier to show a gradient in just one color channel

![esferas-metalicas-01-1](https://user-images.githubusercontent.com/50336793/115978095-a724af80-a553-11eb-834e-3495c34620e9.jpg)

**Comment by rh12503** [comment]
Sorry about the late reply, I thought I'd already left one earlier! 

I understand your desired effect now, but I don't think there is a way to easily implement it, as there are too many factors involved, for example the direction of the gradient and the colors. 

The gradient effect option when exporting would probably be the closest thing you can get. Applying a Gaussian blur in an external editor could work too.

**Comment by jadsongmatos** [comment]
yes, I understand that it would be very complex, maybe an alternative to colors would be to analyze one channel at a time, varying only in intensity

**Comment by rh12503** [comment]
There may be a way to do it, but I feel implementing it is beyond my current abilities. 

I'll be happy to accept a PR from anyone who gets it working!
